taxonomies/ingredients.txt

```diff
@@ -13774,7 +13780,7 @@ kk:Крахмал
 ko:녹말
 la:Amylum
 li:Zètmael
-lt:Krakmolas
+lt:Krakmolas, tirštiklis krakmolas
```

Comment:
is it synonym of Krakmolas or should synonym of "Modifikuotas krakmolas" in additives.txt?

Reply:
tirštiklis is thickener. In one of the product's ingredients it was listed "tirštiklis krakmolas". "**tirštiklis**" should probably be somewhere else not sure where.

Reply:
me neither. Let's leave it in ingredients.txt for now.